docs/beginner/tutorial6-uniforms/README.md

```diff
@@ -230,11 +230,11 @@ Modify the vertex shader to include the following.
 // Vertex shader
 
 [[block]] // 1.
-struct CameraUniform {
+struct Camera {
     view_proj: mat4x4<f32>;
 };
 [[group(1), binding(0)]] // 2.
-var<uniform> camera: CameraUniform;
+var<uniform> camera: Camera;
```

Comment:
I want the name of the struct in WGSL to match the struct in Rust. While not strictly necessary, this makes it a little more clear how they match up.

Reply:
#226 

> @sotrh
At the moment docs/beginner/tutorial6-uniforms/README.md does not match with code/beginner/tutorial6-uniforms/src/shader.wgsl already.
>
> code/ does not raises error while docs/ does. So I didn't change code/ and match docs/ to code/.
>
> Do you mean both definition and usage of Camera in code/**/shader.wgs is incorrect and must be CameraUniform?

Reply:
Yeah the code is incorrect.

Reply:
Basically it should be `CameraUniform` all around.